.github/workflows/rust.yml

```
@@ -141,8 +141,9 @@ jobs:
           forest-cli --chain calibnet db stats
       - name: kill Forest process, print forest logs
         run: |
+          wget -O metrics.log http://localhost:6116/metrics
           pkill forest
-          cat forest.out forest.err
+          cat forest.out forest.err metrics.log
```

Perhaps it'd be easier to navigate the logs if we annotate them a bit
```suggestion
          echo "--- Forest STDOUT ---"; cat forest.out
          echo "--- Forest STDERR ---"; cat forest.err
          echo "--- Forest Prometheus metrics ---"; cat metrics.log
```